.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
```

Comment:
I think we should also add 3.5 most of the distros still using 3.5, what do you think?